Sources/Plasma/Apps/plClient/plClient.cpp

```diff
@@ -2061,8 +2061,8 @@ void plClient::IDetectAudioVideoSettings()
     else
 #endif
     {
-        plPipeline::fDefaultPipeParams.Width = hsG3DDeviceSelector::kDefaultWidth;
-        plPipeline::fDefaultPipeParams.Height = hsG3DDeviceSelector::kDefaultHeight;
```

Comment:
Not sure what was going on here. 'mode' is the default mode - and is already matched against kDefaultWidth and kDefaultHeight within 'GetDefault'